pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -45,9 +45,18 @@ def handle_translation_project(self, translation_project, **options):
 
     def handle_store(self, store, **options):
         overwrite = options.get('overwrite', False)
-        force = options.get('force', False)
-
-        store.update(overwrite=overwrite, only_newer=not force)
+        if not options.get('force', False):
+            disk_mtime = store.get_file_mtime()
+            if not disk_mtime:
```

Comment:
Considering `store.get_file_mtime()` returns `None` when `mtime` is unavailable:
```python
if disk_mtime is None:
```